src/presentation/ui/component/databases/installedDatabaseTypeSummaryTable.templ

```diff
@@ -0,0 +1,56 @@
+package componentDatabases
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+	"strconv"
+)
+
+templ InstalledDatabaseTypeSummaryTable(
+	selectedDatabaseTypeDatabases []entity.Database,
+) {
+	<table class="w-full table-auto border-collapse rounded-md transition-all duration-300 ease-in-out">
+		<thead class="text-xs uppercase text-neutral-400">
+			<tr class="border-b border-neutral-500 border-opacity-90 text-center">
+				<th scope="col" class="px-5 py-3 w-1/7 text-left">Database Name</th>
+				<th scope="col" class="px-5 py-3 w-2/7 text-center">Users</th>
+				<th scope="col" class="px-5 py-3 w-1/7 text-left">Size</th>
+				<th scope="col" class="px-5 py-3 space-x-2 flex justify-end">
+					@componentForm.SubmitButton("Create user", "ph-user-plus", "openCreateDatabaseUserModal()", true)
+					@componentForm.SubmitButton("Create database", "ph-database", "openCreateDatabaseModal()", true)
+				</th>
+			</tr>
+		</thead>
+		<tbody>
+			for _, database := range selectedDatabaseTypeDatabases {
+				<tr class="bg-os-800 text-white">
+					<th scope="row" class="px-5 py-3 w-1/4 text-left font-normal">{ database.Name.String() }</th>
+					<th scope="row" class="px-5 py-3 w-1/4 text-left">
+						<div class="bg-os-500 rounded-md px-2.5 py-1 max-h-75">
+							for _, user := range database.Users {
+								<div class="flex justify-between items-center p-3 my-1.5 bg-os-800 rounded-md">
+									<span class="font-normal">{ user.Username.String() }</span>
+									<div class="flex space-x-2">
+										@componentStructural.CircularIconButtonWithTooltip(
+											"ph-trash", "red-800", "red-600",
+											"openDeleteDatabaseUserModal('"+database.Name.String()+"', '"+user.Username.String()+"')", "Delete user", "red-500",
+										)
+									</div>
+								</div>
+							}
+						</div>
+					</th>
+					<th scope="row" class="px-5 py-3 w-1/4 text-left font-normal">{ strconv.FormatInt(database.Size.Int64(), 10) }</th>
```

Comment:
Interesting. Size is a Byte, right? I'm not sure if it's on OS, but on Control I have lots of getters on Byte so I don't need to use FormatInt and stuff inside the frontend.

Reply:
I added `String()` and used it to cast the content. Thanks for pointing that out.